app/services/businessmatching/BusinessMatchingService.scala

```diff
@@ -92,16 +103,27 @@ class BusinessMatchingService @Inject()(
 
   def getAdditionalBusinessActivities(cacheId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): OptionT[Future, Set[BusinessActivity]] =
     getActivitySet(cacheId, _ diff _)
-
+@deprecated("To be removed when auth implementation is complete")
   def getSubmittedBusinessActivities(implicit ac: AuthContext, hc: HeaderCarrier, ex: ExecutionContext): OptionT[Future, Set[BusinessActivity]] =
     getActivitySet(_ intersect _)
 
+  def getSubmittedBusinessActivities(credId: String)(implicit hc: HeaderCarrier, ex: ExecutionContext): OptionT[Future, Set[BusinessActivity]] =
+    getActivitySet(credId, _ intersect _)
+
+@deprecated("To be removed when wuth implementation is complete")
   def getRemainingBusinessActivities(implicit ac: AuthContext, hc: HeaderCarrier, ec: ExecutionContext): OptionT[Future, Set[BusinessActivity]] =
     for {
       model <- getModel
       activities <- OptionT.fromOption[Future](model.activities)
     } yield BusinessActivities.all diff activities.businessActivities
 
+  def getRemainingBusinessActivities(credId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): OptionT[Future, Set[BusinessActivity]] =
+    for {
+      model <- getModel(credId)
+      activities <- OptionT.fromOption[Future](model.activities)
+    } yield BusinessActivities.all diff activities.businessActivities
+
+  @deprecated("To be removed when new auth is implemented")
   def clearSection(activity: BusinessActivity)(implicit ac: AuthContext, hc: HeaderCarrier) = activity match {
```

Comment:
this method (as example) is used only by this service, so you can delete it and update the specs with new one